README.md

```diff
@@ -4,21 +4,35 @@
 
 This is a generic skeleton project that can be used to quickly get a
 new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started.  This skeleton project
+[Packer](https://packer.io) project started. This skeleton project
 contains [licensing information](LICENSE), as well as
 [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
 appropriate for the major languages that we use.
 
 ## Pre-requisites ##
 
-This project requires a build user to exist in AWS.  The accompanying terraform
-code will create the user with the appropriate name and permissions.  This only
-needs to be run once per project, per AWS account.  This user will also be used by
-GitHub Actions.
+This project requires a build user to exist in AWS. The accompanying Terraform
+code will create the user with the appropriate name and permissions. This only
+needs to be run once per project, per AWS account. This user will also be used
+by GitHub Actions.
+
+Before the build user can be created, the following profile must exist in
+your AWS credentials file:
+
+* `cool-terraform-backend`
+
+The easiest way to set up that profile is to use our
+[`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync) utility.
+Follow the usage instructions in that repository before continuing with the
+next steps. Note that you will need to know where your team stores their
+remote profile data in order to use
+[`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync).
+
+To create the build user, follow these instructions:
 
 ```console
-cd terraform
+cd terraform-test-user
```

Comment:
This line needs to change to `cd terraform-build-user`.

Reply:
Ok I _think_ (_**hope**_) that's all of them now as of 022e5d8.